### ISSUE TITLE
fig_9.ipynb in deJong_et_al_2023 example fails with matplotlib >= 3.8.0

### ISSUE THREAD

**Issue by slayoo** [open-atmos/PySDM#1142]
```
 ran Straub2010 for 32 superdroplets in 37.434369802474976 sec
----- stderr -----
/tmp/ipykernel_5842/3923770933.py:31: MatplotlibDeprecationWarning: The get_cmap function was deprecated in Matplotlib 3.7 and will be removed two minor releases later. Use ``matplotlib.colormaps[name]`` or ``matplotlib.colormaps.get_cmap(obj)`` instead.
  cmap = matplotlib.cm.get_cmap('viridis')
------------------

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[3], line 37
     33 dr = np.concatenate([dr, [dr[-1]]])
     35 legend_entries = []
---> 37 color = next(ax[0]._get_lines.prop_cycler)['color'] # pylint: disable=protected-access
     38 for (j, step) in enumerate(steps):  # pylint: disable=protected-access
     39     if step != 0:

AttributeError: '_process_plot_var_args' object has no attribute 'prop_cycler'
```

**Comment by slayoo** [comment]
addressed in #1235

**Comment by slayoo** [comment]
indeed addressed in #1235, so closing
